PlaidLink/Classes/PLDLinkBankMFAContainerViewController.h

```diff
@@ -24,6 +24,9 @@
 
 @property(nonatomic, weak) id<PLDLinkBankMFAContainerViewControllerDelegate> delegate;
 
-- (instancetype)initWithInstitution:(PLDInstitution *)institution product:(PlaidProduct)product;
+@property(nonatomic, weak) NSDictionary *options;
```

Comment:
It's should be a reference to the property on `PLDLinkNavigationViewController` – since it's just being passed along to the other classes I figured that's better from a memory management standpoint, right?

I'm relatively new to Objective-C so I'm not 100% sure if this is the "correct" way to do it, but it definitely works (it's working in my app, including with a couple of edge cases around no `options`, nil `options, etc)